tests/cisco_ios/show_object-group/cisco_ios_show_object-group-v6.yml

```diff
@@ -0,0 +1,138 @@
+---
+parsed_sample:
+  - ANY: ''
+    DESCRIPTION: ''
+    HOST: ''
+    HOST_RANGE_END: ''
+    HOST_RANGE_START: ''
+    ICMP6_TYPE: ''
+    ICMP_TYPE: ''
+    NAME: TEST-v6-icmp
+    NESTED_GROUPS: ''
+    NETMASK: ''
+    NETWORK: ''
+    PORT: ''
+    PORT_MATCH: ''
+    PORT_RANGE_END: ''
+    PORT_RANGE_START: ''
+    PROTOCOL: ''
+    TYPE: V6-Service
+  - ANY: ''
+    DESCRIPTION: ''
+    HOST: ''
+    HOST_RANGE_END: ''
+    HOST_RANGE_START: ''
+    ICMP6_TYPE: echo-request
+    ICMP_TYPE: ''
+    NAME: TEST-v6-icmp
+    NESTED_GROUPS: ''
+    NETMASK: ''
+    NETWORK: ''
+    PORT: ''
+    PORT_MATCH: ''
+    PORT_RANGE_END: ''
+    PORT_RANGE_START: ''
+    PROTOCOL: ''
+    TYPE: V6-Service
+  - ANY: ''
+    DESCRIPTION: ''
+    HOST: ''
+    HOST_RANGE_END: ''
+    HOST_RANGE_START: ''
+    ICMP6_TYPE: ''
+    ICMP_TYPE: echo-reply
+    NAME: TEST-v6-icmp
+    NESTED_GROUPS: ''
+    NETMASK: ''
+    NETWORK: ''
+    PORT: ''
+    PORT_MATCH: ''
+    PORT_RANGE_END: ''
+    PORT_RANGE_START: ''
+    PROTOCOL: ''
+    TYPE: V6-Service
+  - ANY: ''
+    DESCRIPTION: ''
+    HOST: ''
+    HOST_RANGE_END: ''
+    HOST_RANGE_START: ''
+    ICMP6_TYPE: ''
+    ICMP_TYPE: ''
+    NAME: TEST-v6-obj
+    NESTED_GROUPS: ''
+    NETMASK: ''
+    NETWORK: ''
+    PORT: ''
+    PORT_MATCH: ''
+    PORT_RANGE_END: ''
+    PORT_RANGE_START: ''
+    PROTOCOL: ''
+    TYPE: V6-Network
+  - ANY: ''
+    DESCRIPTION: ''
+    HOST: 2001:DB8::1:1111
+    HOST_RANGE_END: ''
+    HOST_RANGE_START: ''
+    ICMP6_TYPE: ''
+    ICMP_TYPE: ''
+    NAME: TEST-v6-obj
+    NESTED_GROUPS: ''
+    NETMASK: ''
+    NETWORK: ''
+    PORT: ''
+    PORT_MATCH: ''
+    PORT_RANGE_END: ''
+    PORT_RANGE_START: ''
+    PROTOCOL: ''
+    TYPE: V6-Network
+  - ANY: ''
+    DESCRIPTION: ''
+    HOST: 2001:DB8::2:2222
+    HOST_RANGE_END: ''
+    HOST_RANGE_START: ''
+    ICMP6_TYPE: ''
+    ICMP_TYPE: ''
+    NAME: TEST-v6-obj
+    NESTED_GROUPS: ''
+    NETMASK: ''
+    NETWORK: ''
+    PORT: ''
+    PORT_MATCH: ''
+    PORT_RANGE_END: ''
+    PORT_RANGE_START: ''
+    PROTOCOL: ''
+    TYPE: V6-Network
+  - ANY: ''
+    DESCRIPTION: ''
+    HOST: ''
+    HOST_RANGE_END: ''
+    HOST_RANGE_START: ''
+    ICMP6_TYPE: ''
+    ICMP_TYPE: ''
+    NAME: TEST-v6-obj
+    NESTED_GROUPS: ''
+    NETMASK: /48
+    NETWORK: '2001:DB8:111:1::'
+    PORT: ''
+    PORT_MATCH: ''
+    PORT_RANGE_END: ''
+    PORT_RANGE_START: ''
+    PROTOCOL: ''
+    TYPE: V6-Network
+  - ANY: ''
+    DESCRIPTION: ''
+    HOST: ''
+    HOST_RANGE_END: ''
+    HOST_RANGE_START: ''
+    ICMP6_TYPE: ''
+    ICMP_TYPE: ''
+    NAME: TEST-v6-obj
+    NESTED_GROUPS: ''
+    NETMASK: /48
+    NETWORK: '2001:DB8:222:2::'
+    PORT: ''
+    PORT_MATCH: ''
+    PORT_RANGE_END: ''
+    PORT_RANGE_START: ''
+    PROTOCOL: ''
+    TYPE: V6-Network
```

Comment:
```suggestion
    TYPE: V6-Network

```

Reply:
This will be obsoleted by brunopeter/ntc-templates/pull/1